commcare_connect/users/models.py

```diff
@@ -20,7 +20,7 @@ class User(AbstractUser):
     name = models.CharField(_("Name of User"), blank=True, max_length=255)
     first_name = None  # type: ignore
     last_name = None  # type: ignore
-    email = models.EmailField(_("email address"), unique=True)
+    email = models.EmailField(_("email address"))
```

Comment:
We can't make this field not unique, since it is the primary identifier of the user. What is the intent behind this change?

Reply:
I think you need to have `(null=True, blank=True, unique=True)`

Reply:
@snopoke Does that create a partial index on non null values? That is what I was going to recommend.

Reply:
This doesn't create a partial index (though it's not strictly necessary for PostgreSQL since null values are not considered equal. In the long run a partial index would probably be better though.

@pxwxnvermx you can see how to do that here: https://docs.djangoproject.com/en/4.2/ref/models/constraints/#condition

(also remove the `unique=True` kwarg from the column)

Reply:
https://github.com/dimagi/commcare-connect/pull/101/commits/394c1794e8c37ef16679987dd16d4e133c2e4ac0